universal-login-formal-verification/src/spec.md

```diff
@@ -0,0 +1,47 @@
+High-level specification of KeyHolder
+=====================================
+
+-   The function `keyHasPurpose` returns `TRUE` when `#mapping.keys[_key] == _purpose`:
```

Comment:
Shouldn't it be:
The function `keyHasPurpose` returns `TRUE` **if and only if** `#mapping.keys[_key] == _purpose`: